src/app/reducers/index.spec.js

```diff
@@ -8,18 +8,16 @@ import * as actions from './actions';
 
 import reducer, { Helpers as reducerHelpers } from '.';
 
-describe('reducer', function () {
-    // Set up an initial state that can be re-used by the tests
-    before(() => {
-        this.initialState = new Map({
-            puzzle  : reducerHelpers.initPuzzle(puzzleJSON),
-            progress: reducerHelpers.initProgress(puzzleJSON.grid),
-            position: new Map({
-                row: 0,
-                col: 0,
-                dir: 'across'
-            })
-        });
+describe('reducer', () => {
```

Comment:
noice

Reply:
oh btw, all the tests are probably broken right now. I'll fix that before merging